### ISSUE TITLE
Add to documentation attributes that lack functions to set them

### ISSUE THREAD

**Issue by MaxGhenis** [mwburke/stargazer#59]
Per some comments in recent PRs, it sounds like new functions won't be added to set attributes that can be altered directly. Currently, only functions are documented in `README.md` and the example notebook. Attributes that require direct modification should also be documented.

**Comment by toobaz** [comment]
Good point to keep in mind, but #44 will require even deeper changes to the docs.
